pkg/render/tiers/tiers_test.go

```diff
@@ -30,10 +30,15 @@ var _ = Describe("Tiers rendering tests", func() {
 
 	clusterDNSPolicy := testutils.GetExpectedPolicyFromFile("../testutils/expected_policies/dns.json")
 	clusterDNSPolicyForOCP := testutils.GetExpectedPolicyFromFile("../testutils/expected_policies/dns_ocp.json")
+	nodeLocalDNSPolicy := testutils.GetExpectedGlobalPolicyFromFile("../testutils/expected_policies/node_local_dns.json")
 
 	BeforeEach(func() {
 		// Establish default config for test cases to override.
-		cfg = &tiers.Config{Openshift: false}
+		cfg = &tiers.Config{
+			Openshift:    false,
+			NodeLocalDNS: false,
+			KubeDNSCIDR:  "10.96.0.10/32",
```

Comment:
I think these tests will need an update to match the new `tiers.Config` struct.

I'd also suggest adding a dual-stack, as well as an IPv6 single-stack test case here as well.